src/app/main/header/header.component.scss

```diff
@@ -89,4 +113,12 @@
   .action {
     padding: 0;
   }
+  .advanced-search-link {
+      background-color: lavender;
+      margin-top: -100px;
+  }
+  .expert-search-link {
+    background-color: indigo;
```

Comment:
Do you want to keep this color?

Reply:
No, I forgot to remove these classes > done in ec8c3ea